README.md

```diff
@@ -330,6 +370,40 @@ Post.search do
 end
 ```
 
+#### Nested Documents search with `ChildOf` and `ParentWhich`
+
+You can search for child (or parent) documents based on filters applied
+on parents (or children).
+
+##### `ChildOf` operator
```

Comment:
based on above comment: 'Search child documents based on parent properties'.
Same for the latter block